includes/class-assets.php

```diff
@@ -485,4 +488,35 @@ private static function is_jquery_ui_css_loaded() {
 		}
 		return false;
 	}
+
+	/**
+	 * Handles Floating Links' scripts and styles enqueueing.
+	 *
+	 * @since x.x
+	 *
+	 * @param string $plugin_url URL of the plugin.
+	 * @param string $version Current version of the plugin.
+	 * @return void
+	 */
+	private static function enqueue_floating_links( $plugin_url, $version ) {
```

Comment:
Thanks for your work @shervElmi 

I don't think we need to make this function static

Reply:
Thank you for your review, @mostafa-hisham!

We apply Floating Links in all S11 plugins. I made this method static to avoid using any `$this` reference in the future. I believe the static approach works well here.